### ISSUE TITLE
Node Analyzer Benchmark Write Permissions Toggle Statements Duplicated

### ISSUE THREAD

**Issue by dcedstrom** [sysdiglabs/charts#1253]
In the Benchmark Runner Cluster Role there's a toggle based on `.Values.nodeAnalyzer.benchmarkRunner.includeSensitivePermissions` that should be stripping out write permissions required in the role. Above that there is a static statement that gives the Cluster Role for Node Analyzer the permissions regardless. If the value is set to true the same clause shows up twice in the role definition. 

This is the only thing that requires cluster wide pod creation and deletion permissions so if it's not needed I would like to strip these permissions out. If they are needed for Node Analyzer's standard operation, take the toggle out to avoid confusion about if the permissions are necessary.

```yaml
# https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/templates/clusterrole-node-analyzer.yaml#L36
- apiGroups:
    - ""
  resources:
    - "pods"
    - "pods/exec"
  verbs:
    - "create"
- apiGroups:
    - ""
  resources:
    - "pods"
  verbs:
    - "delete"

# https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/templates/clusterrole-node-analyzer.yaml#L99
{{- if .Values.nodeAnalyzer.benchmarkRunner.includeSensitivePermissions  }}
- apiGroups:
    - ""
  resources:
    - pods/exec
  verbs:
    - create
- apiGroups:
    - ""
  resources:
    - pods
  verbs:
    - create
    - delete
{{- end }}
```

**Comment by aroberts87** [comment]
Hey @dcedstrom ! Thanks for raising this issue. @a1ex-var1amov has addressed this issue as a part of #1272 , which has been release for a little while now. Can you confirm that the solution addresses your concerns? Thanks!

**Comment by mavimo** [comment]
@dcedstrom please re-open the discussion if the fix added by @a1ex-var1amov do not satisfy your request!
